packages/runtime/container-runtime/src/test/containerRuntime.spec.ts

```diff
@@ -1295,6 +1295,42 @@ describe("Runtime", () => {
 					"entryPoint does not match expected object",
 				);
 			});
+
+			it("loadRuntime accepts both requestHandlers and entryPoint", async () => {
```

Comment:
I think we want a test that takes only requestHandler and shows that entryPoint is initialized as expected.  It's kind of covered by `when using old load method` today, but I think that will get deleted when the old load method is deleted, we should have one for `loadRuntime` itself.